docs/pages/develop/development-builds/expo-go-to-dev-build.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Convert from Expo Go to a development build
+sidebar_title: Expo Go to development build
+description: How to migrate your Expo Go project to use development builds
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+To migrate from Expo Go to a development build, you'll need to:
+
+<Step label="1">
+
+## Install the `expo-dev-client`
+
+The Expo Dev Client library includes the launcher UI (shown in the screenshots below), dev menu, extensions to test updates, and more. The Expo Go app has the dev menu built in, and that's why you need to install it separately for a development build.
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+<ContentSpotlight
+  alt="expo-dev-client launcher and menu UIs on Android and iOS."
+  src="/static/images/dev-client/preview.png"
+  caption={`When you run a development build it will look like this, only with your app name and icon included rather than "Microfoam". The launcher UI is pictured in iOS on the left and Android on the right. In between, you can see an app running inside of the development build, with the customizable developer menu open.`}
+/>
+
+> We recommend using the `expo-dev-client` for the best development experience, but it is possible to use development builds without installing this library. If not using the dev client, in [Step 3](/develop/development-builds/expo-go-to-dev-build/#start-the-dev-client) start the bundler with `--dev-client`. Otherwise it will default to opening in Expo Go.
+
+</Step>
+
+<Step label="2">
+
+## Build your native app
+
+With Expo Go, we only needed to build the JavaScript bundle, but with development builds we also need to compile the native app. With Expo, there is two parts to building your native app:
+
+1. generate the native `/ios` and/or `/android` directories ([read more](/develop/development-builds/expo-go-to-dev-build/#cng-and-prebuild) on when and how you'll need to do this)
+2. use native build tools to compile the native app(s)
```

Comment:
```suggestion
1. Generate the native **android** and/or **ios** directories ([read more](/develop/development-builds/expo-go-to-dev-build/#cng-and-prebuild) on when and how you'll need to do this)
2. Use native build tools to compile the native app(s)
```

Let's use the platform order as per our writing style guide:https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms